website/content/docs/concepts/policies.mdx

```diff
@@ -800,32 +800,36 @@ authenticated user.
-| [sys/config/cors](/vault/api-docs/system/config-cors)                                                                                                  | GET, PUT, DELETE  | Configure CORS setting                                                                                                                              |
-| [sys/config-ui](/vault/api-docs/system/config-ui)                                                                                                      | GET               | Configure the UI settings                                                                                                                           |
-| [sys/internal/specs/openapi](/vault/api-docs/system/internal-specs-openapi)                                                                           | GET               | Generate an OpenAPI document of the mounted backends                                                                                                |
+| [sys/config/auditing/request-headers/:name](/vault/api-docs/system/config-auditing)                                                                    | GET, POST, DELETE | Manage the auditing headers (create, update, read and delete)                                                                                       |
+| [sys/config/cors](/vault/api-docs/system/config-cors)                                                                                                  | GET, POST, DELETE | Configure CORS setting                                                                                                                              |
+| [sys/config-ui/headers](/vault/api-docs/system/config-ui)                                                                                              | GET               | Configure the UI settings                                                                                                                           |
+| [sys/config-ui/headers/:name](/vault/api-docs/system/config-ui#name)                                                                                   | POST, DELETE      | Configure the values to be returned for the UI header                                                                                               |
+| [sys/internal/inspect/router](/vault/api-docs/system/inspect/router  )                                                                                 | GET               | Inspect the internal components of Vault's router                                                                                                                                      |
 | [sys/leases/lookup/:prefix](/vault/api-docs/system/leases#list-leases)                                                                                 | LIST              | List lease IDs                                                                                                                                      |
```

Comment:
I see your comment:

> Already in table specific to endpoints requiring sudo - sys/leases

But I don't think that's true.

The one existing entry has a path `sys/leases/lookup/:prefix` but a link to `/vault/api-docs/system/leases#list-leases` which is a **different** endpoint!

There are actually three separate similarly named endpoints which should be in the table:

* `sys/leases` https://developer.hashicorp.com/vault/api-docs/system/leases#leases-list (GET)
* `sys/leases/lookup` https://developer.hashicorp.com/vault/api-docs/system/leases#read-lease (POST)
* `sys/leases/lookup/:prefix` https://developer.hashicorp.com/vault/api-docs/system/leases#list-leases (LIST)

All of these actually require sudo capability even though the documentation at the above links doesn't say so for all of them.

Reply:
`/sys/leases/lookup/:prefix` is what is documented at https://developer.hashicorp.com/vault/api-docs/system/leases#list-leases

This endpoint returns a list of lease ids.

This endpoint requires 'sudo' capability.

Method	Path
LIST	/sys/leases/lookup/:prefix

Reply:
Yes, the existing documentation is quite bad at reliably identifying endpoints which require sudo, which is why I believe you should look to the code first instead.

Reply:
Here are the lines of code which mark these endpoints as root-protected:

https://github.com/hashicorp/vault/blob/578e62214ecabdb5e9bd6981085a8a2f71c30cd2/vault/logical_system.go#L119

https://github.com/hashicorp/vault/blob/578e62214ecabdb5e9bd6981085a8a2f71c30cd2/vault/logical_system.go#L121

Reply:
Hi Max,

I will be opening a separate ticket to request help from engineering with this. I am not a developer and do not want to pretend I can interpret the source code properly.